flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontendOptions.java

```diff
@@ -94,6 +94,15 @@ public class CliFrontendOptions {
                                     + "program that was part of the program when the savepoint was triggered.")
                     .build();
 
+    public static final Option FLINK_CONFIG =
+            Option.builder("D")
+                    .longOpt("flink-config")
+                    .hasArg()
+                    .desc(
+                            "<property=value>. Allows specifying multiple flink generic configuration options. The available"
+                                    + "options can be found at https://nightlies.apache.org/flink/flink-docs-stable/ops/config.html")
+                    .build();
```

Comment:
Seems we can declare option like [this](https://github.com/apache/flink/blob/f0c8f18ee09297c4c92b36d8ce7d0e0259f80178/flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliOptionsParser.java#L73-L80) to get rid of the hassle about splitting & parsing KvPairs.